plugin/sampling/strategystore/static/fixtures/probablistic_with_default_operation_strategy.json

```diff
@@ -0,0 +1,14 @@
+{
+    "probabilisticSampling": {
+        "samplingRate": 1
+    },
+    "operationSampling": {
+        "defaultSamplingProbability": 1,
+        "perOperationStrategies": [
+            {
+                "operation": "/health",
+                "probabilisticSampling": {}
```

Comment:
This does not seem like a valid output, where is the rate?

This could be a side effect of JSON marshaling which has an option of omitting fields that have empty value (in this case prob=0). I suggest passing an argument to the marshaler to include empty fields.

Reply:
This is indeed the case. However - from what I see in the suggested fixes on the internet - the options are to either have ProbabilisticSamplingStrategy.SamplingRate declared as `*float64` or remove `omitempty` tag from the field, but as those are generated - does not look like it should be done. 

I was not able to find any property/parameter/option, which I could pass to the marshaler/unmarshaler to control this behavior (I might be missing something too, as this is literally the first time I write something in GO :) )

So took the easy way out and added non-zero values to the test, so now the operation level strategy sampling probability values are there and are verified. 

I also checked how the configuration is received on Java app side - it does have 0.0 probability if that is the value set in strategy configuration and accordingly sampling does not happen for the identified operation. So maybe its ok to leave the tests with 0.1?